lib/rules/block-spacing.js

```diff
@@ -15,7 +15,8 @@ var util = require("../ast-utils");
 module.exports = function(context) {
     var always = (context.options[0] !== "never"),
         message = always ? "Requires a space" : "Unexpected space(s)",
-        sourceCode = context.getSourceCode();
+        sourceCode = context.getSourceCode(),
+        tokensAndComments = sourceCode.tokensAndComments;
```

Comment:
We can't change what `tokensAndComments` is at this point. It needs to stay an array.